utils/customize_node.py

```diff
@@ -188,9 +201,11 @@ def set_apt_proxy(args):
     # insufficient.  Either way, get the proxies from os.environ.
     path = args.new_fs_dir + '/etc/apt/apt.conf.d'
     file_utils.make_dir(path)
-    path += '/00FAMproxy.conf'  # !!! FIXME: no hardcoded custom file names !!!
+    path += '/00TMMS.conf'
     args.apt_dot_conf = path    # for post-processing
-    if not args.is_golden:      # One final post-processing step is done later
+
+    # One final post-processing step is done later
+    if not args.hostname == 'golden':
```

Comment:
This is a BAD BAD BAD way to do this, it's why I pulled it out of the last round.  There are many places the state needs to be known and a string literal comparison in each is cumbersome.

Reply:
Never compare literal strings in multiple places.  It's bad style in general and violated Python DRY specifically.   You have to keep track of every location it's done, you have to know that the magic values are (what if there are more).  Put back the boolean field the way I had it.   If it was only one instance I'd consider it but there are half a dozen and it may not be finished growing.

Reply:
Alright, I can't counter that one. I'll get back to bool. For the record, I don't like my approach nor I like the extra bool in args.